examples/jsm/nodes/accessors/TangentNode.js

```diff
@@ -97,7 +97,7 @@ export const tangentGeometry = nodeImmutable( TangentNode, TangentNode.GEOMETRY
 export const tangentLocal = nodeImmutable( TangentNode, TangentNode.LOCAL );
 export const tangentView = nodeImmutable( TangentNode, TangentNode.VIEW );
 export const tangentWorld = nodeImmutable( TangentNode, TangentNode.WORLD );
-export const transformedTangentView = label( tangentView, 'TransformedTangentView' );
+export const transformedTangentView = temp( tangentView, 'TransformedTangentView' );
```

Comment:
I think `temp()`  would be responsible for creating a temporary variable, with a name or anonymous (automatic). `label()` must force the assignment of a name, usually related to bindings: uniform/textures/...

Reply:
Yes, I think that's what I mentioned above about with a name or anonymous... Seems easier to use `node.temp( 'myVar' )` instead of `node.temp().label( 'myVar' )`. `label()` now doesn't create variables anymore, it just helps to define a name when it is declared.